.github/workflows/ci-ptf.yml

```diff
@@ -51,6 +51,7 @@ jobs:
       - name: Build (Ubuntu 20.04)
         run: |
           tools/ci-build.sh
+          sudo pip3 install scapy==2.5.0
```

Comment:
This is the wrong location, the ci build script should have a section for these kinds of tests (PTF_PSA?)